pkg/storage.go

```diff
@@ -130,6 +130,8 @@ type Filesystem struct {
 	Name string
 	// FsType of the Filesystem
 	FsType string
+	// DiskType of the Filesystem
+	DiskType string
```

Comment:
Do we really need this type, because as far as i remember, we have a VolumeType defined somewhere with probably exactly possible values.

Reply:
I think we do yes, only this Filesystem is exposed with zbus

Reply:
I don't get your answer

Reply:
Are you talking about this: https://github.com/threefoldtech/zos/blob/736c1119840e06aa6919935826c7abc01c8ddcf3/pkg/storage/filesystem/filesystem.go#L17 ?

Reply:
@muhamadazmy ?

Reply:
https://github.com/threefoldtech/zos/blob/736c1119840e06aa6919935826c7abc01c8ddcf3/pkg/storage.go#L47